lldb/include/lldb/Target/Language.h

```diff
@@ -160,6 +166,10 @@ class Language : public PluginInterface {
 
   virtual lldb::LanguageType GetLanguageType() const = 0;
 
+  // Implement this function to return the user-defined entry point name
+  // for the language
+  virtual const char *GetUserEntryPointName() const { return nullptr; }
```

Comment:
nit: this could have returned an `llvm::StringRef` instead of an `const char *`.